activities/management/commands/jslps_hnn_person_new1.py

```diff
@@ -0,0 +1,147 @@
+import urllib2
+import unicodecsv as csv
+import xml.etree.ElementTree as ET
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+from geographies.models import *
+from people.models import *
+from programs.models import *
+import jslps_data_integration as jslps
+
+class Command(BaseCommand):
+	def handle(self, *args, **options):
+		#read xml from url
+		url = urllib2.urlopen('http://webservicesri.swalekha.in/Service.asmx/GetExportGroupMemberDataHnNNew1?pUsername=admin&pPassword=JSLPSSRI')
```

Comment:
We can use string manipulation to change the url to remove all extraneous files. Or we can send the url as argument to call_command method

Reply:
Let me try this